test/scheduled_skills.py

```diff
@@ -0,0 +1,29 @@
+from datetime import datetime, timedelta
+import unittest
+
+from mycroft.skills.scheduled_skills import ScheduledSkill
+from mycroft.util.log import getLogger
+
+__author__ = 'eward'
+
+logger = getLogger(__name__)
+
+
+class ScheduledSkillTest(unittest.TestCase):
+    def test_formatted_time_today_hours(self):
+        date = datetime.now() + timedelta(hours=2)
+        self.assertEquals(ScheduledSkill(name='test').
```

Comment:
Create only one ScheduledSkill obj inside __init__ for your test class and then reuse the object.
```
def __init__():
  self.skill = ScheduledSkill('ScheduledSkillTest')
...
self.assertEquals(self.skill.get_formatted_time(...))
```